### PR TITLE
Test funder fail

### LINKED ISSUE #15: Add failing funder test
Go-Perun now has a generic failing funder test. This test should also be integrated here.

### PR DESCRIPTION
**Pull request by matthiasgeihs**
This PR integrated the failing funder test.

Closes #15

### CODE REVIEW

File: channel/pallet/funder_test.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func TestFunder_Timeout(t *testing.T) {
 	// Bob did not fund and times out.
 	wantErr := makeTimeoutErr(1)
 	// Only call Alice's funder.
-	ctx, cancel := context.WithTimeout(context.Background(), 10*s.BlockTime)
+	ctx, cancel := context.WithTimeout(context.Background(), 15*s.BlockTime)
```
Comment (cryptphil) [inline channel/pallet/funder_test.go:68]: Maybe use a constant? Also makes it easier to adapt it afterward.

File: client/appchannel_test.go [hunk 2]
```diff
@@ -55,8 +55,7 @@ func TestAppChannel(t *testing.T) {
 		),
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), TestTimeoutBlocks*s.BlockTime)
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
```
Comment (cryptphil) [inline client/appchannel_test.go:58]: Same here. Similar to the TwoPartyTestTimeout in go-perun?

**Review by cryptphil — APPROVED** [review]
Looks good to me, just added comments regarding the timeouts which could be defined as constants.

File: channel/pallet/funder_test.go (hunk 1 above)
Comment (matthiasgeihs) [reply channel/pallet/funder_test.go]: Reverted the change. It was actually not a problem.

File: client/appchannel_test.go (hunk 2 above)
Comment (matthiasgeihs) [reply client/appchannel_test.go:58]: I don't want to use a global timeout. Each test has a different running time.

**Pull request merged**